src/libsyntax/diagnostic_list.rs

```diff
@@ -16,31 +16,394 @@
 register_long_diagnostics! {
 
 E0533: r##"
+The export_name attribute was malformed.
```

Comment:
Shouldn’t you put things like export_name inside inline code element (i.e. between backticks)?